server/data/mappers/workAndInterestMapper.ts

```diff
@@ -65,6 +65,11 @@ const toWorkExperience = (ciagInduction: CiagInduction): WorkExperience => {
 }
 
 const toLongQuestionSetWorkInterests = (ciagInduction: CiagInduction): WorkInterests => {
+  const mostRecentlyUpdatedSection: 'MAIN_INDUCTION' | 'WORK_INTERESTS' =
+    ciagInduction.modifiedDateTime >= ciagInduction.workExperience.workInterests.modifiedDateTime
```

Comment:
This is ok, but personally I would have found an if statement quicker to grasp. It's a shame typescript doesn't have when statements!